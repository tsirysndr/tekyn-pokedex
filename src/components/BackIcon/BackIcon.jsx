import * as React from "react";
import Svg, { Path } from "react-native-svg";

const BackIcon = (props) => (
  <Svg
    width={18}
    height={18}
    viewBox="0 0 24 40"
    fill="none"
    xmlns="http://www.w3.org/2000/svg"
    {...props}
  >
    <Path
      d="M1.129 17.157c.203-.202.423-.373.655-.521L17.286 1.134a3.876 3.876 0 0 1 5.48 5.48L9.413 19.964l13.42 13.422a3.874 3.874 0 0 1-5.478 5.479L1.784 23.29a3.85 3.85 0 0 1-.655-.522A3.85 3.85 0 0 1 0 19.963a3.852 3.852 0 0 1 1.128-2.806Z"
      fill="#264653"
    />
  </Svg>
);

export default BackIcon;
