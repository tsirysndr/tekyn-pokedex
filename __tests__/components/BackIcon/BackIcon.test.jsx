import BackIcon from "@Components/BackIcon";
import { render } from "@testing-library/react-native";

test("renders the BackIcon", () => {
  const tree = render(<BackIcon />);
  expect(tree).toMatchSnapshot();
});
