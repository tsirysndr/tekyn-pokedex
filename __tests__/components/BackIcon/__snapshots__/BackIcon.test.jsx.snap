// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the BackIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight={18}
  bbWidth={18}
  fill="none"
  focusable={false}
  height={18}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      Object {
        "flex": 0,
        "height": 18,
        "width": 18,
      },
    ]
  }
  vbHeight={40}
  vbWidth={24}
  width={18}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      Array [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M1.129 17.157c.203-.202.423-.373.655-.521L17.286 1.134a3.876 3.876 0 0 1 5.48 5.48L9.413 19.964l13.42 13.422a3.874 3.874 0 0 1-5.478 5.479L1.784 23.29a3.85 3.85 0 0 1-.655-.522A3.85 3.85 0 0 1 0 19.963a3.852 3.852 0 0 1 1.128-2.806Z"
      fill={4280698451}
      propList={
        Array [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
